e2e/lib/k8up.bash

```diff
@@ -126,12 +129,34 @@ given_an_annotated_subject() {
 	export BACKUP_FILE_NAME=${1}
 	export BACKUP_FILE_CONTENT=${2}
 
-	kubectl apply -f definitions/pv
+	kubectl apply -f definitions/pv/pvc.yaml
 	yq e '.spec.template.spec.containers[0].securityContext.runAsUser='$(id -u)' | .spec.template.spec.containers[0].env[0].value=strenv(BACKUP_FILE_CONTENT) | .spec.template.spec.containers[0].env[1].value=strenv(BACKUP_FILE_NAME)' definitions/annotated-subject/deployment.yaml | kubectl apply -f -
 
 	echo "✅  The annotated subject is ready"
 }
 
+given_a_rwo_pvc_subject_in_worker_node() {
+	require_args 2 ${#}
+
+	export BACKUP_FILE_NAME=${1}
+	export BACKUP_FILE_CONTENT=${2}
+
+	yq e 'with(select(document_index == 2) .spec.template.spec; .containers[0].env[0].value=strenv(BACKUP_FILE_CONTENT) | .containers[0].env[1].value=strenv(BACKUP_FILE_NAME))' definitions/pvc-rwo-subject/worker.yaml | kubectl apply -f -
+
+	echo "✅  The pvc rwo worker subject is ready"
+}
+
+given_a_rwo_pvc_subject_in_controlplane_node() {
+	require_args 2 ${#}
+
+	export BACKUP_FILE_NAME=${1}
+	export BACKUP_FILE_CONTENT=${2}
+
+	yq e 'with(select(document_index == 2) .spec.template.spec; .containers[0].securityContext.runAsUser='$(id -u)' | .containers[0].env[0].value=strenv(BACKUP_FILE_CONTENT) | .containers[0].env[1].value=strenv(BACKUP_FILE_NAME))' definitions/pvc-rwo-subject/controlplane.yaml | kubectl apply -f -
```

Comment:
Is there a reason you run the control plane with `runAsUser` and the worker one not? This leads to a permission denied on my mac...

Reply:
good catch - not intentional, was an oversight. I'll double check everything on macOS again. :+1: